Documentation/building/cross-building.md

```diff
@@ -1,7 +1,23 @@
-Cross Compilation for ARM
-=========================
+Cross Compilation for ARM on Linux
+==================================
+
+Through cross compilation, on Linux it is possible to build CoreCLR for arm or arm64.
```

Comment:
To be more consistent with the argument provided which are arm and arm64. Someone else might want to weight in there.